.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: node_js
+sudo: false
+node_js:
+- '4.2.3'
+
+addons:
+  firefox: latest
+
+cache:
+  directories:
+    - node_modules
+
+env:
+  global:
+  - ARCH=linux-x64
+
+install:
+  - npm install
+
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+
+script:
+  - ./node_modules/.bin/karma start --single-run --browsers Firefox
```

Comment:
Yeah, but Firefox is easier on Travis. I originally set it up with Chrome, but there was a cryptic runtime error that I couldn't reproduce locally on my linux box. I'll open an issue to set up Browser Stack for more browsers.